api_tests/nodes/views/test_node_links_detail.py

```diff
@@ -125,7 +125,7 @@ def test_cannot_delete_if_registration(self):
             pointer_id,
         )
         res = self.app.delete(url, auth=self.user.auth, expect_errors=True)
-        assert_equal(res.status_code, 403)
+        assert_equal(res.status_code, 404)
```

Comment:
I think the reason deleting is not currently giving a 405 is because the RegistrationNodeLinksDetail view inherits from the NodeLinksDetail view, which inherits from RetrieveDestroyAPIView.  So Destroy is implemented here, however, the permission_class "ReadOnlyIfRegistration" is probably what's giving the 403.

If you want to get a 405 I would not inherit from the NodeLinksDetail view.